searx/user_help.py

```diff
@@ -37,12 +46,16 @@ def render(app: flask.Flask):
 
     define_link_targets = ''.join(f'[{name}]: {url}\n' for name, url in link_targets.items())
 
-    for filename in pkg_resources.resource_listdir(__name__, 'help'):
-        rootname, ext = os.path.splitext(filename)
-
-        if ext != '.md':
-            continue
-
-        markdown = pkg_resources.resource_string(__name__, 'help/' + filename).decode()
-        markdown = define_link_targets + markdown
-        HELP[rootname] = mistletoe.markdown(markdown)
+    for pagename in _TOC:
```

Comment:
In which case a file exists in `searx/help` but is not declared in `_TOC`?
Instead:
* keep `pkg_resources.resource_listdir(__name__, 'help')`
* pass the `_TOC` values to `help.html` templates, so only `{% for name, page in all_pages %}` can be `{% for name, page in toc_pages %}`

But I may misunderstood the intent

Reply:
If a file would exist in `searx/help` without being listed in `_TOC` it would simply be ignored, which as far as I'm concerned isn't a problem. Do you find it problematic?

Reply:
> Do you find it problematic?

Nope.